src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php

```diff
@@ -342,7 +340,7 @@ private function getFormWithFields(ElementInterface $element, array $fields)
             }
         }
 
-        return null;
+        return;
```

Comment:
Can we leave `return null`? IMHO if method returns some value, it should use `return $something` and `return null`, if method does not return anything, it should use `return`.